src/main.ts

```diff
@@ -432,15 +432,15 @@ class Benchmark extends utils.Adapter {
 
 	/**
 	 * Measure the Node.js event loop lag and repeatedly call the provided callback function with the updated results
-	 * @param {number} ms The number of milliseconds for monitoring
-	 * @param {function} cb Callback function to call for each new value
+	 * @param ms The number of milliseconds for monitoring
+	 * @param cb Callback function to call for each new value
 	 */
-	private measureEventLoopLag(ms:number, cb: (lag: number) => void): void {
+	private measureEventLoopLag(ms:number, cb?: (lag: number) => void): void {
```

Comment:
`cb` was optional, so it should be marked optional

Reply:
was optional because of copy paste (optional means there was a cb && check)